pkg/kubelet/kuberuntime/kuberuntime_container.go

```diff
@@ -594,6 +615,13 @@ func (m *kubeGenericRuntimeManager) killContainer(pod *v1.Pod, containerID kubec
 		glog.V(3).Infof("Container %q exited normally", containerID.String())
 	}
 
+	if m.containerManager != nil {
+		hdlr := m.containerManager.GetDevicePluginHandler()
+		if hdlr != nil {
+			hdlr.DeallocateDevices(pod, containerName)
```

Comment:
It seems that it does not call for func in client to release the device in the realization of DeallocateDevices， like it does in func AllocateDevices, can the devices get release gracefully?